plugin.video.dumpert/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+	id="plugin.video.dumpert"
+	name="Dumpert"
+	version="1.1.10+matrix.1"
+	provider-name="Skipmode A1">
+  <requires>
+    <import addon="xbmc.python"                     version="3.0.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.3.2"/>
+    <import addon="script.module.requests"          version="2.4.3"/>
+    <import addon="script.module.future"            version="0.0.1"/>
+  	<import addon="script.module.html5lib"          version="0.999.0"/>
+    <import addon="plugin.video.youtube"            version="5.1.7"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" 	library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Watch funny videos from Dumpert.nl (dutch)</summary>
+        <description lang="en_GB">Watch funny videos from Dumpert.nl (dutch)</description>
+        <disclaimer lang="en_GB">For bugs, requests or general questions visit the Dumpert.nl thread on the Kodi forum.</disclaimer>
+    <summary lang="nl_NL">Bekijk grappige filmpjes van Dumpert.nl (dutch)</summary>
+        <description lang="nl_NL">Bekijk grappige filmpjes van Dumpert.nl (dutch)</description>
+        <disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de Dumpert.nl thread op het Kodi forum.</disclaimer>
+    <language>nl</language>
+    <platform>all</platform>
+    <license>GPL-2.0-or-later</license>
```

Comment:
The same issue with the license here. It should be "GPL-3.0" as your LICENSE.txt has "GNU GENERAL PUBLIC LICENSE 3, 29 June 2007"

Reply:
Fixed